src/plugins/tabletools.js

```diff
@@ -366,17 +366,19 @@
 
 		let rowsToDelete = [];
 
+		let rows;
+
 		// Figure out selected cells' column indices.
-		for (var i = 0, rows = map.length; i < rows; i++) {
-			for (var j = 0, cols = map[i].length; j < cols; j++) {
+		for (let i = 0, rows = map.length; i < rows; i++) {
+			for (let j = 0, cols = map[i].length; j < cols; j++) {
```

Comment:
Note this `cols` doesn't suffer the same problem as `rows` in the line above. `cols` is only accessed in the loop condition and not outside of it.